packages/html/testing-library.js

```diff
@@ -0,0 +1,3 @@
+// NOTE: Use this file as a fallback for utils those don't support `exports` field in package.json
+/** @type {import('./src/testing-library')} */
+module.exports = require("./dist/cjs/testing-library");
```

Comment:
This is ok if it works. We just need to test and make sure that it doesn't change the behavior of the libraries that _do_ support package exports. In other words, we don't want it to overwrite all the cases (especially an ESM bundler).